app/namespace.go

```diff
@@ -108,16 +108,28 @@ var (
 		Usage:   "The codec server endpoint to decode payloads for all users interacting with this Namespace, must be https",
 		Aliases: []string{"e"},
 	}
-	serviceAccountPrincipalFlag = &cli.StringFlag{
+	saPrincipalFlagRequired = &cli.StringFlag{
 		Name:     "service-account-principal",
-		Usage:    "service account that has access to the sink",
+		Usage:    "Service account that has access to the sink",
 		Required: true,
 	}
-	gcsBucketFlag = &cli.StringFlag{
+	saPrincipalFlagOptional = &cli.StringFlag{
+		Name:  "service-account-principal",
+		Usage: "Service account that has access to the sink",
+	}
+	gcsBucketFlagRequired = &cli.StringFlag{
 		Name:     "gcs-bucket",
-		Usage:    "gcs bucket of the sink",
+		Usage:    "GCS bucket of the sink",
 		Required: true,
 	}
+	enableCmekFlag = &cli.BoolFlag{
+		Name:  "enable-cmek",
+		Usage: "Enable customized encryption.  Note: If enable cmek needs to be updated, user should create the service account with cmek or modify the created service account accordingly. Provided it as part of the input won't help",
```

Comment:
[Nit] the last sentence of this usage statement doesn't really provide any extra information/guidance. can we remove this?

let's also replace user "should" create the service account with cmek with user "must"